test/functional/test-custom-element.js fails when run under chai 4

Chai 4 isn't checked in yet, but I'm going to have to disable part of this test to let the rest continue to run after the chai upgrade.
```
  CustomElement
     
      ● should apply heights condition
        Chrome 61.0.3163 (Linux 0.0.0)
      AssertionError: expected '1%' to equal '99%'
          at Context.<anonymous> (/tmp/test/functional/test-custom-element.js:904:54 <- /tmp/3b7bf95e5ec46ec2526522ce3ad9fb49.browserify:217855:55)
```
See #11390

--- a/extensions/amp-access/0.1/test/test-amp-access.js
+++ b/extensions/amp-access/0.1/test/test-amp-access.js
@@ -541,6 +541,7 @@ describes.fakeWin('AccessService authorization', {
 
     service.analyticsEvent_ = sandbox.spy();
     performanceMock = sandbox.mock(service.performance_);
+    performanceMock.expects('onload_').atLeast(0);
   });
 
   afterEach(() => {

--- a/extensions/amp-access/0.1/test/test-amp-access.js
+++ b/extensions/amp-access/0.1/test/test-amp-access.js
@@ -744,8 +744,7 @@ describes.fakeWin('AccessService authorization', {
     });
   });
 
-  // TODO(dvoytenko): Enable after fixing #11398.
-  it.skip('should resolve first-authorization promise after success', () => {
+  it('should resolve first-authorization promise after success', () => {
     expectGetReaderId('reader1');
     adapterMock.expects('authorize')
         .withExactArgs()

--- a/test/functional/test-custom-element.js
+++ b/test/functional/test-custom-element.js
@@ -892,8 +892,7 @@ describes.realWin('CustomElement', {amp: true}, env => {
     expect(element2.style.width).to.equal('50vw');
   });
 
-  // TODO(dvoytenko): Enable after fixing #11400.
-  it.skip('should apply heights condition', () => {
+  it('should apply heights condition', () => {
     const element1 = new ElementClass();
     element1.sizerElement = doc.createElement('div');
     element1.setAttribute('layout', 'responsive');

